x-pack/solutions/security/plugins/security_solution/public/asset_inventory/sample_data.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { type DataTableRecord } from '@kbn/discover-utils/types';
+
```

Comment:
it occurred to me that it might be better to leverage [ES Archiver](https://docs.elastic.dev/kibana-dev-docs/ops/es-archiver) to add the mock data directly into mapped ES indices, so we can have the functionalities that require interaction with Elasticsearch working (sort datagrid, search, filter, etc), just for a follow up PRs, no action needed for this PR.